owncloud/core
# Exclude custom data directory from integrity checker

We should not scan any custom data directory in the integrity checker as well. Otherwise this would lead to a massive increased update time that may likely exceed the timeout.

To test this:
1. Install ownCloud 8.2.2, set as data dir another folder than `data` such as `data-asdf`. Make sure that folder lives in the web root as well.
2. Update that instance to 9.0.0 beta2 => Errors are shown in the code integrity check
3. Apply this patch and trigger the rescan in the admin settings => Only one errors is shown and this is this file (can also do a new update with that patch applied)

Fixes https://github.com/owncloud/core/issues/22698
